codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/protocol/ServerProtocolTestGenerator.kt

```diff
@@ -525,14 +525,14 @@ class ServerProtocolTestGenerator(
             FailingTest(RestJson, "RestJsonSupportsNegativeInfinityFloatInputs", Action.Response),
             FailingTest(RestJson, "RestJsonStreamingTraitsWithBlob", Action.Request),
             FailingTest(RestJson, "RestJsonStreamingTraitsWithNoBlobBody", Action.Request),
-            FailingTest(RestJson, "RestJsonStreamingTraitsWithBlob", Action.Response),
```

Comment:
@david-perez @crisidev I was not sure what to do here - should I remove these tests from the expect-fail list as these now seem to pass...?

Reply:
Yep, if it passes it can be removed from the failing list.

Reply:
I manually verified the now-passing tests and they look good.

I also had a look at the remaining failing request streaming protocol tests and pushed a new commit to this branch, [Add blob streaming support for server request protocol tests](https://github.com/awslabs/smithy-rs/pull/1023/commits/0033f7e942b4d37beb6ceb671ce9b1c20749608c), to get rid of them.

The only remaining failing protocol test for streaming is `RestJsonStreamingTraitsRequireLengthWithBlob`, but that is due to us not setting `Content-Length` appropriately, so I've cut https://github.com/awslabs/smithy-rs/issues/1146 to track that.

Let me know if you agree with the fix or have any questions.

Reply:
That's great @david-perez. Thank you for making it so easy!